src/VisualStudio/CSharp/Impl/ProjectSystemShim/HACK_VariantStructure.cs

```diff
@@ -23,18 +23,17 @@ namespace Microsoft.VisualStudio.LanguageServices.CSharp.ProjectSystemShim
     /// as this structure. We can then pick out this broken pattern, and convert
     /// it to null instead of true.
     /// </summary>
+    [StructLayout(LayoutKind.Auto)]
```

Comment:
IDE0051 handles StructLayout attribute and doesn't produce diagnostics when it exists. But I'm not sure of the correct usage of the attribute.

Reply:
```suggestion
    [StructLayout(LayoutKind.Sequential)]
```